lxd/api_internal.go

```diff
@@ -122,14 +122,8 @@ func internalImport(d *Daemon, r *http.Request) Response {
 		return BadRequest(fmt.Errorf("target is required"))
 	}
 
-	path := containerPath(name, false)
-	err := d.Storage.ContainerStart(name, path)
-	if err != nil {
-		return SmartError(err)
-	}
-
-	defer d.Storage.ContainerStop(name, path)
-
+	// COMMENT(brauner): Check if we received a storage pool on which the
+	// storage volume for the container is supposed to be created.
```

Comment:
Hmm, what?

I see code being removed and a comment added but no code to match the comment :)